ckanext/switzerland/templates/robots.txt

```diff
@@ -0,0 +1,4 @@
+User-agent: *
+{% block all_user_agents -%}
```

Comment:
AFAIK the syntax for a block is `{% block here_comes_the_block_name %}`
Can you double-check the syntax here and in the `{%- endblock %}`-line? 
If I remember it correctly, it should be possible to acess the robots.txt locally, too

Reply:
I copied the template over from ckan and just changed the line in the block, see here: https://github.com/ckan/ckan/blob/2.8/ckan/templates/robots.txt

Reply:
okay, did not remember that. if it works for you locally, I think we can merge it :)

Reply:
Okay thanks.